src/ai/backend/manager/idle.py

```diff
@@ -126,22 +132,25 @@ async def _do_idle_check(
         source: AgentId,
         event: DoIdleCheckEvent,
     ) -> None:
-        log.debug('do_idle_check(): triggered')
+        log.debug("do_idle_check({}, {}s): triggered", self.name, str(self.check_period))
         async with self._db.begin() as conn:
```

Comment:
Let's replace `begin()` with `begin_readonly()` because idle checkers will only read the database.